extension/package.json

```diff
@@ -423,6 +423,12 @@
         "category": "DVC",
         "icon": "$(symbol-class)"
       },
+      {
+        "title": "Add Plot",
```

Comment:
Maybe "Add Top-Level Plot" to better differentiate between "Add Custom Plot"?

Reply:
This is ok for now.